homeassistant/components/spotify/coordinator.py

```diff
@@ -42,6 +43,7 @@ class SpotifyCoordinator(DataUpdateCoordinator[SpotifyCoordinatorData]):
     """Class to manage fetching Spotify data."""
 
     current_user: UserProfile
+    config_entry: ConfigEntry
```

Comment:
Is this initialized somewhere?

Reply:
In the coordinator constructor

Reply:
Should this not be a typed config entry?
```suggestion
    config_entry: SpotifyConfigEntry
```

Reply:
I don't really use the runtime data here though

Reply:
> I don't really use the runtime data here though

I thought the policy was to update it everywhere...
For example, even if a migration doesn't use runtime_data, we still recommend the signature to match.